compiler/rustc_middle/src/ty/context.rs

```diff
@@ -472,12 +472,7 @@ impl<'tcx> TyCtxtFeed<'tcx, LocalDefId> {
     }
 }
 
-/// The central data structure of the compiler. It stores references
```

Comment:
Hm, I feel like `GlobalContext` is a bit of an implementation detail unless you're working on the query system.

At the least, this should link via a doc comment to gcx so people can click to see the more useful doc comment on that instead. Ideally, though, this should explain that the arenas and queries are accessible via tcx as well.

Reply:
I moved the comment back to `TyCtxt` and tweaked it accordingly.